src/app/(tabs)/_layout.tsx

```diff
@@ -1,11 +1,12 @@
 import { Tabs } from "expo-router";
+import Home_icon from "assets/icons/Home_icon.svg";
```

Comment:
let's rename these to follow the component naming convention

HomeIcon ... HomeIcon.svg

ArrowLeftIcon ... ArrowLeftIcon.svg

Reply:
Those where only examples, I already got the SVG's from Designer and we will implement all ths SVG according to that name convention

Reply:
I did a new commit with all the SVG and the name according to the conventions